.gitignore

```diff
@@ -31,4 +32,8 @@ tmp/
 pkg/
 dist/
 node_modules/
-.parcel-cache/
\ No newline at end of file
+.parcel-cache/
+web/node_modules
+package-lock.json
+web/package-lock.json
+web/etc/proxy/package-lock.json
```

Comment:
Please remove all added `package-lock.json` and `package.json` files. Then remove these lines from `.gitignore`.